client/lib/purchases/index.js

```diff
@@ -213,7 +213,11 @@ function showCreditCardExpiringWarning( purchase ) {
 }
 
 function showEditPaymentDetails( purchase ) {
-	return ! isExpired( purchase ) && ! isOneTimePurchase( purchase ) && ! isIncludedWithPlan( purchase ) && isPaidWithCreditCard( purchase );
+	return showEditPaymentDetailsOption( purchase ) && isPaidWithCreditCard( purchase );
+}
+
+function showEditPaymentDetailsOption( purchase ) {
```

Comment:
> I'm having trouble understanding why it is necessary to have both of these functions. Could you explain it?

They are used for two different cases:
- Deciding when to show the edit payment details link
- Deciding when to show the payment details themselves

We need to show the edit payment details link even if we don't have credit card details, but we don't want to show the credit card details on the manage purchase screen if we don't have any credit card data.